openslides_backend/shared/mixins/user_scope_mixin.py

```diff
@@ -160,7 +176,137 @@ def check_permissions_for_scope(
+                )
+
+    def check_for_admin_in_all_meetings(
+        self,
+        instance_or_id: int | dict[str, Any],
+        b_meeting_ids: set[int] | None = None,
+    ) -> bool:
+        """
+        Checks if the requesting user has permissions to manage participants in all of requested users meetings but requested user doesn't.
+        Also checks requested user has no committee management rights if a dict was provided instead of an id.
+        Returns true if permissions are given. False if not. Raises no Exceptions.
+        """
+        if isinstance(instance_or_id, dict):
+            if not (instance_id := instance_or_id.get("id")):
+                return False
+            if hasattr(self, "name") and self.name == "user.create":
+                if instance_or_id.get("committee_management_ids", []):
+                    return False
+            else:
+                user = self.datastore.get(
+                    fqid_from_collection_and_id("user", instance_id),
+                    ["committee_management_ids"],
+                    lock_result=False,
+                    use_changed_models=False,
+                )
+                if user.get("committee_management_ids", []):
+                    return False
+        else:
+            instance_id = instance_or_id
+        if not b_meeting_ids:
+            if not hasattr(self, "instance_committee_meeting_ids"):
+                return False
+            b_meeting_ids = set()
+            for m_ids in self.instance_committee_meeting_ids.values():
+                if m_ids:
+                    b_meeting_ids.update(m_ids)
+        if not b_meeting_ids:
+            return False
+        # During participant import there is no permstore.
+        if hasattr(self, "permstore"):
+            a_meeting_ids = (
+                self.permstore.user_meetings
+            )  # returns only admin level meetings
+        else:
+            a_user = self.datastore.get(
+                fqid_from_collection_and_id("user", instance_id),
+                ["meeting_ids"],
+                lock_result=False,
+            )
+            a_meeting_ids = set(a_user.get("meeting_ids", []))
+        if not a_meeting_ids:
+            return False
+        intersection_meeting_ids = a_meeting_ids.intersection(b_meeting_ids)
+        if not b_meeting_ids.issubset(intersection_meeting_ids):
+            return False
+        intersection_meetings = self.datastore.get_many(
+            [
+                GetManyRequest(
+                    "meeting",
+                    list(intersection_meeting_ids),
+                    ["admin_group_id", "group_ids", "meeting_user_ids"],
+                )
+            ],
+            lock_result=False,
+        ).get("meeting", {})
+        for meeting_id, meeting_dict in intersection_meetings.items():
+            admin_meeting_users = {}
+            admin_group_id = meeting_dict.get("admin_group_id", 0)
```

Comment:
Every meeting has an `admin_group_id`. It is a required field. If the tests will fail with `meeting_dict["admin_group_id"]` that's a sign that the tests are wrong

Reply:
Did so and updated the tests.